gateway/gateway.go

```diff
@@ -439,10 +436,7 @@ func (gw *Gateway) modifyUsername(msg config.Message, dest *bridge.Bridge) strin
 }
 
 func (gw *Gateway) modifyAvatar(msg config.Message, dest *bridge.Bridge) string {
-	iconurl := gw.BridgeValues().General.IconURL
-	if iconurl == "" {
-		iconurl = dest.GetString("IconURL")
```

Comment:
Note that this actually changes behaviour, but I don't think it was intentional thought: IT used to be that if icon url was set in general, it wouldn't be checked in this account, but now it only gets general if account value not set